### PR TITLE
Bump asciidoctor-reveal.js to branch master

### PR DESCRIPTION
**Pull request by jtama**
Following a discussion with @ggrossetie, the published docs should follow master branch.

### CODE REVIEW

**Comment by mojavelinux** [comment]
It looks like there is a missing attribute. See https://app.netlify.com/sites/asciidoctor-docs/deploys/6751772f2da350000829abdc for details.

**Comment by jtama** [comment]
it says it's missing attribute: `url-gh-ggrosetie` -> https://github.com/asciidoctor/asciidoctor-reveal.js/blob/master/docs/antora.yml#L33

**Comment by mojavelinux** [comment]
That's correct. The reference is spelled wrong.

**Comment by mojavelinux** [comment]
Also, the version in antora.yml should match the version that's released (in this case 5.2). This is why the recommended behavior is to always create a branch for each minor version. That way, the docs and the release match. (In theory, patch releases should not impact the documentation). And the docs in the main/master branch for the next minor or major release can be modified without impacting the documentation.

In summary, there should be a v5.2.x branch that corresponds to the 5.2.0 (and later patches) release that is used in the documentation site. (You can see we do this for Asciidoctor and Asciidoctor PDF).

**Comment by ggrossetie** [comment]
> Also, the version in antora.yml should match the version that's released (in this case 5.2). This is why the recommended behavior is to always create a branch for each minor version. That way, the docs and the release match. (In theory, patch releases should not impact the documentation). And the docs in the main/master branch for the next minor or major release can be modified without impacting the documentation.

We think that we can reduce the maintenance effort by versioning the documentation only for major versions.
So, the main branch (master) will be "5" and the maintenance branch will be "4". The main drawback is that the documentation might describe a feature not yet released but we feel like a maintenance per minor versions is a bit much for this project.

**Comment by mojavelinux** [comment]
That's certainly your choice and judgement call. And if it works, it works. I have found that versioning by minor version avoids the problem that the published documentation gets ahead of what is released. But that's just my $0.02.

You are always free to remove old branches once they are not needed anymore. In other words, keep a v5.2.x branch and a master branch. Then when 5.3.0 is released, remove the v5.2.x branch and add a v5.3.x branch. That should keep maintenance to a minimum.

**Comment by jtama** [comment]
Changes have been made, on the asciidoctor-revealjs repository, could it be possible to trigger the checks ?

**Comment by mojavelinux** [comment]
I just reran the build and it succeeded. For some reason, it's not updating here, though. But we can still move forward.
